packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -78,6 +79,12 @@ const PersonalDetailsForm = props => {
 
     const is_rendered_for_idv_or_onfido = is_rendered_for_idv || is_rendered_for_onfido;
 
+    const is_poi_verified =
+        has_real_account &&
+        [AUTH_STATUS_CODES.PENDING, AUTH_STATUS_CODES.VERIFIED].includes(
+            account_status?.authentication?.identity.status
```

Comment:
just a doubt, do we need optional chaining at identity as well?
`?.identity?.status`

Reply:
Yeh.. Makes sense. Will make this change